moduleframework/common.py

```diff
@@ -800,39 +738,39 @@ def get_config(fail_without_url=True, reload=False):
         cfgfile = os.environ.get('CONFIG')
         if cfgfile:
             if os.path.exists(cfgfile):
-                print_debug("Config file defined via envvar: %s" % cfgfile)
+                core.print_debug("Config file defined via envvar: %s" % cfgfile)
             else:
-                raise ConfigExc("File does not exist although defined CONFIG envvar: %s" % cfgfile)
+                raise mtfexceptions.ConfigExc("File does not exist although defined CONFIG envvar: %s" % cfgfile)
         else:
             cfgfile = "./config.yaml"
             if os.path.exists(cfgfile):
-                print_debug("Using module config file: %s" % cfgfile)
+                core.print_debug("Using module config file: %s" % cfgfile)
             else:
                 if fail_without_url and not get_url():
-                    raise DefaultConfigExc("You have to use URL envvar for testing your images or repos")
+                    raise mtfexceptions.Defaultmtfexceptions.ConfigExc("You have to use URL envvar for testing your images or repos")
                 cfgfile = "/usr/share/moduleframework/docs/example-config-minimal.yaml"
-                print_debug("Using default minimal config: %s" % cfgfile)
+                core.print_debug("Using default minimal config: %s" % cfgfile)
 
 
         try:
             with open(cfgfile, 'r') as ymlfile:
                 xcfg = yaml.load(ymlfile.read())
             doc_name = ['modularity-testing', 'meta-test-family', 'meta-test']
```

Comment:
Why `meta-test`?

Reply:
I don't know. I've defided it looong time ago, as shorter name in config files, it is closed to modularity-testing, it also miss word framework